app/models/participant_profile/ecf.rb

```diff
@@ -52,17 +52,15 @@ def self.ransackable_associations(_auth_object = nil)
     %w[cohort participant_identity school user teacher_profile induction_records]
   end
 
-  def self.unfinished_with_billable_declaration(cohort:, restrict_to_participant_ids: [])
-    billable_states = %w[eligible payable paid].freeze
-
-    return none unless cohort == Cohort.active_registration_cohort
+  def self.unfinished_with_billable_declaration(cohort: Cohort.active_registration_cohort, restrict_to_participant_ids: [])
+    return none if cohort.blank?
```

Comment:
```suggestion
    return none unless cohort.start_year == 2024
```

Reply:
this is going to be rebased from leandro's pr so hopefully we will resolve there